frontend/src/components/GroupView/MiddleBar/index.tsx

```diff
@@ -1,25 +1,43 @@
 import React, { ReactElement } from 'react';
-import { Group, SamwiseUserProfile } from 'common/types/store-types';
+import { Group, SamwiseUserProfile, Task } from 'common/types/store-types';
 
 import GroupViewMiddleBarPeopleList from './GroupViewMiddleBarPeopleList';
 import GroupViewMiddleBarTaskQueue from './GroupViewMiddleBarTaskQueue';
 import styles from './index.module.scss';
 
 type Props = {
   readonly group: Group;
-  readonly groupMemberProfiles: readonly SamwiseUserProfile[];
+  readonly groupMemberProfiles: SamwiseUserProfile[];
   readonly changeView: (selectedGroup: string | undefined) => void;
+  readonly tasks: readonly Task[];
 };
 
-const GroupViewMiddleBar = ({ groupMemberProfiles, group, changeView }: Props): ReactElement => (
-  <div className={styles.MiddleBar}>
-    <GroupViewMiddleBarTaskQueue />
-    <GroupViewMiddleBarPeopleList
-      group={group}
-      groupMemberProfiles={groupMemberProfiles}
-      changeView={changeView}
-    />
-  </div>
-);
+const GroupViewMiddleBar = ({
+  groupMemberProfiles,
+  group,
+  changeView,
+  tasks,
+}: Props): ReactElement => {
+  const allInfo: any = localStorage.getItem('firebaseui::rememberedAccounts');
+  const allInfoJSON = JSON.parse(allInfo);
+  const { email } = allInfoJSON[0];
```

Comment:
do not use this to get user. We have a better typed version. See

https://github.com/cornell-dti/samwise/blob/72aeb1ddc75401244758d090f9c0c1f8d400807c/frontend/src/firebase/auth-util.ts#L40-L48